src/cpp/jank/runtime/obj/transient_vector.cpp

```diff
@@ -0,0 +1,118 @@
+#include "jank/runtime/detail/object_util.hpp"
+#include "jank/runtime/erasure.hpp"
+#include "jank/type.hpp"
+#include <jank/runtime/util.hpp>
+#include <jank/runtime/obj/native_function_wrapper.hpp>
+#include <jank/runtime/obj/transient_vector.hpp>
+#include <stdexcept>
```

Comment:
I'd be surprised if this is needed, but, if it is, please put it first, separated from the jank includes with an empty line.

Reply:
It's the LSP, lol